client/blocks/signup-form/p2.scss

```diff
@@ -1,133 +0,0 @@
-@import "calypso/assets/stylesheets/p2-extends";
-
-.is-p2-invite.invite-accept,
```

Comment:
Is the p2-specific invite flow another area that we can clean up? I know we have a dedicated set of component specifically for handling that flow for p2s.